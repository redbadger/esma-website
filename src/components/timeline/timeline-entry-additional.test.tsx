import React from "react";
import renderer from "react-test-renderer";
import TimelineEntryAdditional from "./timeline-entry-additional";

describe("TimelineEntryAdditional", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <TimelineEntryAdditional />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});