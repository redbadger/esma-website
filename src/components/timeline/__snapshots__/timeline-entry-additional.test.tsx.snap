// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TimelineEntryAdditional renders correctly 1`] = `
<div
  className="css-17iig0m-timelineEntryAdditionalStyles"
>
  <test-file-stub />
  <h2>
    Why social mobility is better for everyone
  </h2>
  <p>
    For decades, research on educational inequality has concentrated on the school sector where there has been a striking and persistent attainment gap between children from lower and higher socio-economic groups.
  </p>
  <p>
    In more recent years, the evidence base has grown to expose the reproduction of inequality in higher education and the labour market.
  </p>
  <p>
    This has heightened the need for joined-up, cross-sector and collaborative approaches to respond to the scale of the challenge.
  </p>
  <button
    className="css-lmjzsp-darkStyles"
    onClick={[Function]}
  >
    Read research
  </button>
</div>
`;
